### ISSUE TITLE
Notification checkbox not appearing

### ISSUE THREAD

**Issue by adammontandon** [stcr/subscribe-to-comments-reloaded#625]
I am using Astra theme and Learndash plugin, with comments on my learndash lesson pages.

When I install STCR when I am logged in as admin I see "You can manage the subscriptions of this post." displayed right above the "post comment" button. But I do NOT see the checkbox for notifications. When I log in as another user (non-admin) nothing shows at all. What am I missing?

**Comment by BobaWebDev** [comment]
Hi @adammontandon 

Can you send over the URL to the site, seeing it might give me an idea of what's going on.

**Comment by tdechangy** [comment]
I had the exact same issue, using Astra too so I tested with Twenty twenty-one, giving the same result.
Note that I see the "Manage subscriptions link", but no checkbox(es).
Unfortunately I am now on localhost so I have to wait until it's live to share an URL.

Did you find your solution @adammontandon ?

Hoping to solve this asap since it's a quite essential plugin :-D

**Comment by BobaWebDev** [comment]
Hi @tdechangy 

The user @adammontandon had the issue where he was seeing the "manage subscriptions" link when he's logged in as administrator but nothing showed when he was logged out, it should show a checkbox for visitors to subscribe.

Seeing the "manage subscriptions" when logged in as the administrator is correct. 

What are you seeing when you visit that post while being logged out?

**Comment by tdechangy** [comment]
You are right, I didn't notice this strange behaviour. I finally found the related option.
Can you just tell why an admin won't be allowed for any reason to see those subscription fields ? Appart of creating this "issue" I can't find any reason in fact.
Thx already for the quick answer !

**Comment by adammontandon** [comment]
Hi, sorry for the delay, I have been busy over the holiday period.
Right now I can't send over an example because my site is behind a paywall, but I will try to set up an empty example page so that you can see what is going on.

**Comment by BobaWebDev** [comment]
Hi @tdechangy 

By default the admin isn't shown the option to subscribe since WordPress already notifies the admin on new comments.

But you can enable the subscription for admins as well, go to WP admin > StCR > Options. Around the middle of the options you'll see "Let Admin Subscribe", set that to "Yes".

@adammontandon 

No worries, happy holidays.

**Comment by adammontandon** [comment]
Hi @BobaWebDev 
I have made a test so that you can see the problem I am having.

Go to www.speakersimpact.com and click "Login" in the upper right hand corner.

Log in with username: commenttest password: testcomment
This is a user with the role of "Subscriber"

If you go to this page: https://speakersimpact.com/test-page-1/
You will see it works as expected with "Notify me of follow up comments via E-mail" correctly displayed. - This is a normal "page" within wordpress.

However, if you go to  this page: https://speakersimpact.com/lessons/2-07-brainstorming-external-values/
You will see that the "Notify me of follow up comments via E-mail" is not displayed in the comments area.
This page is a Learndash "focus mode" with astra theme.

Any ideas?

**Comment by adammontandon** [comment]
I am looking at it in more depth now and I think it is a css dispaly:none setting.... somewhere.....

**Comment by adammontandon** [comment]
it turns out I can fix the problem with it not displaying in Learndash Focus Mode by clicking on Appearance > Customise and then in the Additional CSS section adding the lines:

label[for=subscribe-reloaded] {
	display:block !important;
            }

**Comment by BobaWebDev** [comment]
The theme might have `label { display: none; }` for some reason. Might be some plugin as well, but it's unlikely. 

Do a search for the term `label` in the style.css of the theme.
